homework/english/raby_alexandr/lection1/vocabulary.md

```diff
@@ -0,0 +1,17 @@
+1. I boot Win10 before work.
```

Comment:
Booted. If u say about past time

Reply:
We study the Present Simple and Present Continuous. Why use the Past Simple?

Reply:
Then u should correctly construct the sentence and add keywords where it is necessary